_data/sites.json

```
@@ -10907,9 +10907,9 @@
 
     {
         "name": "Pixilart",
-        "url": "https://www.pixilart.com/",
-        "difficulty": "impossible",
-        "notes": "You can deactivate your account in 'Account Settings' -> 'Deactivate Account', but you can't completely delete it",
+        "url": "https://www.pixilart.com/settings/delete",
+        "difficulty": "easy",
+        "notes": "Fairly simple process but can be hard to find the delete link",
```

```suggestion
        "notes": "Fairly simple process, but can be hard to find the delete link",
```
Doesn't the link take directly to the delete button/page? If so I think we might want to omit these notes.

---

yeah, true, I just copied it from the issue, but might be better to change it